3-fetch-api-ajax/fetch.js

```diff
@@ -0,0 +1,20 @@
+'use strict'
+
+const ARTICLE = document.querySelector('.main-article');
+const AUTHOR_IMG = ARTICLE.appendChild(document.createElement('img'));
+const AUTHOR_NAME = ARTICLE.appendChild(document.createElement('h3'));
```

Comment:
```suggestion
const article = document.querySelector('.main-article');
const author_img= article.appendChild(document.createElement('img'));
const author_name = article.appendChild(document.createElement('h3'));
```
Upper case `const` are more commonly used for static values.